.github/workflows/doc-build.yml

```diff
@@ -102,6 +102,8 @@ jobs:
         run: |
           set -ex
           export SAGE_USE_CDNS=yes
+          export SAGE_LIVE_DOC=yes
+          export SAGE_JUPYTER_SERVER=binder:sagemath/sage-binder-env/dev
           mv /sage/local/share/doc/sage/html/en/.git /sage/.git-doc
```

Comment:
I think we should activate it in the doc-build workflow only on pushes to release tags.

Elsewhere, in particular in a PR, I think it is problematic that the live doc uses a Sage version that is unrelated to the tested branch!

Reply:
Right. 

I will change this.

Reply:
How we will incorporate live doc to our ci (if ever) will be discussed separately. So I will remove the two lines.

But as this PR now needs review, I will keep the lines for now. If it is decided to merge this PR, then I will immediately push a commit to remove them.

Reply:
To conditionalize its use in "build.yml" , one could use `$GITHUB_REF ` https://docs.github.com/en/actions/learn-github-actions/variables